com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Fixed
 
+- Fixed issue where in-scene placed `NetworkObject`s with complex nested children `NetworkObject`s (more than one child in depth) would not synchronize properly if WorldPositionStays was set to true. (#2796)
```

Comment:
Please move this into a new [Unreleased] section since it won't be part of 1.8.0 <3

Reply:
👍 
Will do.
Once develop is updated with the final v1.8.0 changelog file, I will just migrate this entry into the [Unreleased] section.

Reply:
Fixed this issue.
The changelog entry now resides in an unreleased section.